### PR TITLE
MVSNet Release/0.1.0 -- integration pt 1

### PR DESCRIPTION
**Pull request by cpheinrich**
This PR rewires import statements and updates setup.py so that MVSNet can be easily installed and run with pip. To install this branch you would run
```
pip install git+https://github.com/ubiquity6/MVSNet@release/0.1.0
```
It also includes some updates to the inference script. In particular, `mvsnet_pipeline` has been changed to `compute_depth_maps` and the interface has been updated so there is more flexibility when this function is imported elsewhere.

Note that there is still some cleanup work left to do in the test.py script, and repo in general, and I have another Jira ticket to track that work. However I don't expect the interface to change for the compute_depth_maps function so we can be free to use this elsewhere

### CODE REVIEW

File: mvsnet/test.py [hunk 1]
```diff
@@ -66,16 +64,18 @@
 FLAGS = tf.app.flags.FLAGS
 
 
-def mvsnet_pipeline(test_folder, mvs_list=None):
-    """ mvsnet in altizure pipeline """
+def compute_depth_maps(input_dir, output_dir = None, width = None, height = None):
```
Comment (cpheinrich) [inline mvsnet/test.py:69]: This is the function we will call from inside Atlas to compute depth maps

File: mvsnet/test.py [hunk 2]
```diff
@@ -1,46 +1,44 @@
...
+
+logger = mu.setup_logger('mvsnet-inference')
 sys.path.append("../")
 
 # dataset parameters
 tf.app.flags.DEFINE_string('dense_folder', None,
                            """Root path to dense folder.""")
 tf.app.flags.DEFINE_string('model_dir',
-                           None,
+                           'gs://mvs-training-mlengine/trained-models/06-04-2019/',
```
Comment (cpheinrich) [inline mvsnet/test.py:32]: This is the best trained model we have at the moment

Comment (cpheinrich) [reply mvsnet/test.py:32]: although note that since this is pulled from a google storage bucket it will be easy to update

**Review by miaoshasha — APPROVED** [review]
lgtm

**Pull request merged**